Vagrantfile

```diff
@@ -6,24 +6,14 @@ Vagrant.configure("2") do |config|
 
   # This is the dev VM which is going to be provisioned. Only add basic stuff here!
   # Installation and configuration is to be done via playbook
-  config.vm.define "devVM" do |devVM|
-    devVM.vm.network :private_network, ip: "192.168.33.13"
-    devVM.vm.hostname = "devVM"
-    devVM.vm.synced_folder ".", "/vagrant", disabled: true
+  config.vm.define "devvm" do |devvm|
+    devvm.vm.network :private_network, ip: "192.168.33.13"
```

Comment:
As of now we probably don't need the static ip anymore. @p0wnbauer what do you think?